.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Test
+on:
+  pull_request:
+    branches:
+      - '**'
+    types: [ opened, synchronize, reopened ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install Python dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install -r requirements.txt
+        shell: bash
```

Comment:
Where is setup python in here?

Reply:
Fixed in commit de20ffe11d225bd2de36bdbe64f82debc8110561.